Source/Core/VideoBackends/D3D/PSTextureEncoder.cpp

```diff
@@ -26,6 +26,8 @@ struct EFBEncodeParams
   DWORD SrcTop;
   DWORD DestWidth;
   DWORD ScaleFactor;
+  float y_scale;
+  DWORD padding[3];
```

Comment:
DWORD is a windows-ism. Though it's in D3D code, I still think it's better to stick with platform-agnostic types.

Reply:
Agreed but feel like this is better left to another PR where it can change all the encode params.

Reply:
I opened PR #6153 , depending on which gets merged first, I'll update the other.